docker/recorder/src/test/java/org/arquillian/cube/docker/impl/client/reporter/TakeDockerEnvironmentTest.java

```diff
@@ -131,4 +160,48 @@ public void should_report_schema_of_docker_composition() {
         assertThat(screenshotEntry.getLink()).isEqualTo("reports/schemas/docker_composition.png");
     }
 
+   @Test
+    public void shouldRepostLogFile(){
```

Comment:
We should stick to one test method naming convention. Either camel case or underscore. I would opt for _